src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -1,16 +1,18 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System;
+#nullable enable
+
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.IO;
+#if UNIX
```

Comment:
I don't see below why this is only for UNIX?

Reply:
It is from old code. Removed.

Reply:
Ah, it is still needed - line 249 (Unix logging).